LiskHQ/lisk-hub
# Cache delegate names lookup in local storage - Closes #783

### What was the problem?
Delegates in `Delegate vote` transaction type were fetching all delegate usernames by sending for each API request
### How did I fix it?
Added cache so it saves delegates data when user scrools delegate list
### How to test it?
Go to Delegate and scroll down a bit then Find `Delegate vote` transaction, open it and check if it makes additional requests.
### Review checklist
- The PR solves #783
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
